packages/storybook8/stories/Feedback.mdx

```diff
@@ -0,0 +1,60 @@
+import { DefaultButton } from '@fluentui/react';
+import { Meta } from '@storybook/addon-docs';
+import { addCSS } from './utils';
+
+<Meta title="Feedback" />
+
+<div id='feedback-page'>
+<script> { 
```

Comment:
Just a suggestion, maybe we can try to load and register icons in the script tag here?